docroot/profiles/custom/cgov_site/src/Plugin/FeaturesAssignment/CgovFeaturesAlter.php

```diff
@@ -0,0 +1,55 @@
+<?php
```

Comment:
Cgov_alter FeatureAssignment - Allows us to alter the configurations being exported from features to remove the dependencies we do not want. In this case, we remove the dependencies on entities so that workflow can be imported and than configured per-entity if/when that entity is enabled.